configure.ac

```diff
@@ -74,6 +74,12 @@ fi
 AM_CONDITIONAL(ENABLE_TESTS_GLES,
     [test "x$enable_tests_gles" = "xyes"])
 
+HAVE_GTEST="no"
+ifdef([GTEST_LIB_CHECK],
```

Comment:
Agree.  And we should make that clear in the main README for this project if that is what we expect from contributing developers.  Of course, it will also require full support from the project maintainers/gatekeepers to ensure this guideline is followed.  Hopefully that won't be an issue :smile: